shared/cfg/guest-os/Linux/RHEL/7.2/ppc64.cfg

```diff
@@ -15,3 +15,6 @@
         cdrom_cd1 = isos/linux/RHEL-7.2-Server-ppc64.iso
         md5sum_cd1 = d4e597629de04a848c124d3589472372
         md5sum_1m_cd1 = 8c3e5a70edb328f58887c99fc4c4654b
+    unattended_install.url:
+        #Running from http server, RHEL requires more memory
+        mem = 4096
```

Comment:
Seems that this change can be moved to upper cfg file (`7.2.cfg`), for removing redundancies.